pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -128,4 +128,13 @@ under the License.
             <resolver>org.apache.hawq.pxf.plugins.gemfirexd.GemFireXDResolver</resolver>
         </plugins>
     </profile>
+    <profile>
+        <name>Json</name>
+        <description>A profile for JSON data, one JSON record per line</description>
```

Comment:
Thanks for spotting this @GodenYao! Just fixed it

P.S. as an experiment i've implemented a Json Resolver that leverages [JsonPath](https://github.com/jayway/JsonPath). It allows you to  traverse arbitrary complex json records. (current expression mechanism is quite limited). But this an experimental work and we should leave a possible future improvement. Perhaps as an additional 3rd mode -  the most flexible but potentially slowest.